status-reporter/status_reporter/postgresql_reporter.py

```diff
@@ -55,16 +55,20 @@ def _get_patroni_status(self):
         if not detailed_status:
             return NodeServiceStatus.INACTIVE, {}
 
-        if detailed_status['state'] != 'running':
-            return NodeServiceStatus.INACTIVE, detailed_status
+        replication = detailed_status.get('replication', [])
+        replication = [{'sync_state': rep['sync_state'],
```

Comment:
'replication' seems to contain data about replications, so I think it needs a better name